docs/planning/ROADMAP.md

```
@@ -132,4 +244,4 @@ See [GitHub Issues](https://github.com/RackulaLives/Rackula/issues) for ways to
 ---
 
 _This document defines product vision and the active plan. For live work items, see
-[GitHub Milestones](https://github.com/RackulaLives/Rackula/milestones)._
+GitHub Milestones._
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)